packages/@angular/cli/utilities/stats.ts

```diff
@@ -41,7 +39,18 @@ export function statsToString(json: any, statsConfig: any) {
         .join('');
 
       return `chunk {${y(chunk.id)}} ${g(files)}${names}${size} ${initial}${flags}`;
-    }).join('\n')}
+    }).join('\n');
+
+  const unchangedChunkNumber = json.chunks.filter((chunk: any) => !chunk.rendered).length;
```

Comment:
`const unchangedChunkNumber = json.chunks.length - changedChunksStats.length;`

Reply:
nvm didn't see the `join`. Could still keep the resulting array and join later.

Reply:
That's a good idea, I'll do that.